### DISCUSSION TITLE
希望你們可以儘快改善tshet-uinh-extrapolation-tool，以有助分析字音

### DISCUSSION THREAD

**Discussion by 307587** [nk2028/qieyun-extrapolation-tools#4] [Ideas]
我就是用tshet-uinh-extrapolation-tool來分析字音，但其功能有限，只有分類沒有統計功能，例如用同一音韻地位各字的不同讀音計算「不規則讀音」比例。
建議加入「統計同地位演變成不同音」的功能，聲韻調能分開計更好。（當然前提是字音表本身要分聲韻調三欄）

**Comment by ayaka14732** [comment]
主要是有多音字，比較麻煩
